web/static/js/components/shadowed_scroll_container.jsx

```diff
@@ -1,22 +1,23 @@
-import React, { PropTypes } from "react"
-import styles from "./css_modules/shadowed_scroll_container.css"
```

Comment:
ahhhh. we actually can't get rid of this line. i know it showed up in the linter due to `styles` not being referenced, but the `import` statement ensures that the global styles therein are added to the stylesheet.

To fix the issue, we could A) disable the rule for this line, or B) just scope the styles to the `ShadowScrollContainer` component by stripping the `:global` off the `.overlay` styles in the stylesheet, and referencing `className={styles.overlay}` on line 5 below.

Actually thinking about it, the latter is definitely what we'll want to go with, as I can totally see a dev coming along and creating a global class of "overlay". by scoping it with css module hashes, we'll be protected from that scenario